charts/prometheus-ipmi-exporter/templates/configmap.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if not .Values.configMapFile -}}
+{{- if not .Values.useSecret -}}
```

Comment:
I'd suggest a differently named toggle (more below):
```suggestion
{{- if not .Values.configSecret.enabled -}}
```

Reply:
Thanks, makes sense and so I've amended the PR as you've suggested